src/screens/Settings.js

```diff
@@ -187,6 +207,17 @@ class Settings extends React.Component {
           <h4>{t`Advanced Settings`}</h4>
           <div className="d-flex flex-column align-items-start mt-4">
             <p><strong>{t`Allow notifications:`}</strong> {this.state.isNotificationOn ? <span>{t`Yes`}</span> : <span>{t`No`}</span>} <a className='ml-3' href="true" onClick={this.toggleNotificationSettings}> {t`Change`} </a></p>
+            <p>
+              <strong>{t`Hide zero-balance tokens:`}</strong> {
+              this.state.zeroBalanceTokensHidden
+                ? <span>{t`Yes`}</span>
+                : <span>{t`No`}</span>
+              }
+              <a className="ml-3" href="true" onClick={this.toggleZeroBalanceTokens}> {t`Change`} </a>
+              <i className="fa fa-question-circle pointer ml-3"
+                 title={t`When selected, any tokens with a balance of zero will not be displayed anywhere in the wallets.`}>
```

Comment:
Did you run `make update_pot` to add this new string to locale? We should also make sure that it is translated to portuguese

Reply:
Done on [6cfa920](https://github.com/HathorNetwork/hathor-wallet/pull/309/commits/6cfa9204e4de33859e0dabb7892b62cb5ae244e9)